makeflow/src/makeflow_amazon_batch_setup

```diff
@@ -0,0 +1,179 @@
+#!/bin/sh
+#
+#Copyright (C) 2018- The University of Notre Dame
+#This software is distributed under the GNU General Public License.
+#See the file COPYING for details.
+#
+
+
+load_fieldINI()
+{
+	grep $1 $2 | cut -d "=" -f 2 | tr -d ' '
+}
+
+load_fieldJSON()
+{
+	grep $1 $2 | cut -d ":" -f 2 | tr -d ' ' | tr -d ','
+}
+
+if [ -z "$1" ]; then
+	echo "Empty number of CPUs requested."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 32
```

Comment:
Usually we use the same exit code for the same kind of thing. So, a missing argument would be something like exit 1 You don't need exit 32, 33, 34 as all of the errors are about missing arguments.